napari/layers/utils/layer_utils.py

```diff
@@ -47,8 +47,9 @@ def _inner(func):
         )
         if isinstance(shortcuts, str):
             shortcuts = [shortcuts]
-        for shortcut in shortcuts:
-            action_manager.bind_shortcut(name, shortcut)
+        if shortcuts:
+            for shortcut in shortcuts:
```

Comment:
I think that @Carreau made `bind_shortcut` take a list or a string?

Reply:
@goanpeca It takes a string.  `register_layer_action` will take a list[str] or string and then in the registration (this code), it will bind each individual shortcut string within the list.  I didn't actually change this code, except to skip binding if shortcuts was None.

Reply:
👍🏼

Reply:
Some tests are failing though 🤔

Reply:
Yep, will look at that.  :)